realm/src/main/java/io/realm/internal/Row.java

```diff
@@ -108,8 +109,164 @@
 
     /**
      * Returns {@code true} if the field name exists.
+     *
      * @param fieldName Field name to check.
      * @return {@code true} if field name exists, {@code false} otherwise.
      */
     boolean hasColumn(String fieldName);
+
+    Row EMPTY_ROW = new Row() {
+        private final static String UNLOADED_ROW_MESSAGE = "Can't modify an unloaded row, make sure the instance" +
```

Comment:
"an row that hasn't been loaded" or " an row that are still being loading" provides more information IMO